index-live.md

```diff
@@ -89,7 +89,7 @@ margin: auto;
       <p class="textcard">Jupyter lab for Python, R and Julia</p>
     </a>
 	
-    <a class="mcard" href="https://live.usegalaxy.eu/?tool_id=interactive_ml_tool_jupyter_notebook" target="_top">
+    <a class="mcard" href="https://live.usegalaxy.eu/?tool_id=interactive_tool_ml_jupyter_notebook" target="_top">
```

Comment:
but then we need to change the tool as well: https://live.usegalaxy.eu/?tool_id=interactive_tool_ml_jupyter_notebook does not work at the moment.

Reply:
It's coming
https://github.com/usegalaxy-eu/galaxy/commit/5eb2447deb9f7825eb1d27f468d0135e37919c26